#! /bin/sh /usr/share/dpatch/dpatch-run
## apps_Makefile.dpatch by Jose Carlos Garcia Sogo <jsogo@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Cleanup other subdirs Makefiles

@DPATCH@
diff -urNad asterisk-1.0.3/apps/Makefile /tmp/dpep.FJ3Q8L/asterisk-1.0.3/apps/Makefile
--- asterisk-1.0.3/apps/Makefile	2005-01-15 18:28:29.000000000 +0100
+++ /tmp/dpep.FJ3Q8L/asterisk-1.0.3/apps/Makefile	2005-01-15 18:28:29.000000000 +0100
@@ -90,7 +90,7 @@
 	$(CC) $(SOLINK) -o $@ $< -lodbc
 
 look:	look.c
-	$(CC) -pipe -O6 -g look.c -o look -lncurses
+	$(CC) -pipe -g look.c -o look -lncurses
 
 ifneq ($(wildcard .depend),)
 include .depend
diff -urNad asterisk-1.0.3/codecs/lpc10/Makefile /tmp/dpep.FJ3Q8L/asterisk-1.0.3/codecs/lpc10/Makefile
--- asterisk-1.0.3/codecs/lpc10/Makefile	2005-01-15 18:28:29.000000000 +0100
+++ /tmp/dpep.FJ3Q8L/asterisk-1.0.3/codecs/lpc10/Makefile	2005-01-15 18:28:29.000000000 +0100
@@ -39,7 +39,7 @@
 ifeq ($(PROC),ultrasparc)
         CFLAGS+= -mtune=$(PROC) -mcpu=v8 -O3 -fomit-frame-pointer
 else
-        CFLAGS+= -march=$(PROC)
+	CFLAGS+=$(shell if $(CC) -march=$(PROC) -S -o /dev/null -xc /dev/null >/dev/null 2>&1; then echo "-march=$(PROC)"; fi)
 endif
 endif
 endif
@@ -47,7 +47,7 @@
 endif
 endif
 
-LIB = $(LIB_TARGET_DIR)/liblpc10.a
+LIB = ./liblpc10.a
 
 .PHONY: all clean
 
